src/components/ui/PriceDisplay.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class PriceDisplay extends React.Component {
+    static propTypes = {
+        fish: PropTypes.object.isRequired,
+        professionMultiplier: PropTypes.number.isRequired,
+    };
+
+    profMult = (price) => {
+        return (
```

Comment:
Since the only thing this arrow function is doing is returning, it shouldn't be in braces and it should be on a single line.